test/remote/gateways/remote_elavon_test.rb

```diff
@@ -40,7 +40,7 @@ def test_authorize_and_capture
     assert_equal 'APPROVAL', auth.message
     assert auth.authorization
 
-    assert capture = @gateway.capture(@amount, auth.authorization, credit_card: @credit_card)
+    assert capture = @gateway.capture(@amount, auth.authorization, @options.merge(credit_card: @credit_card))
```

Comment:
This change was made to pass in the address fields to this `capture` request to get this previously failing test to pass now that it's receiving the `avs_address` field as part of this request.